failed to run ./test-performance.sh multiple times

<!-- This form is for bug reports and feature requests ONLY!

If you're looking for help check [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes) and the [troubleshooting guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/).

If the matter is security related, please disclose it privately via https://kubernetes.io/security/.
-->

**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind bug

> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind feature


**What happened**:
There are two issues：

1. When you run ./test-performance.sh multiple times in succession, a test failure will be reported

```
W0609 08:31:53.210004   24243 factory.go:1345] Request for pod sample/sched-perf-pod-62wtd already in flight, abandoning
E0609 08:31:53.212949   24243 factory.go:1323] Error scheduling sample sched-perf-pod-bqszw: Post http://127.0.0.1:58132/api/v1/namespaces/sample/pods/sched-perf-pod-bqszw/binding: read tcp 127.0.0.1:62863->127.0.0.1:58132: read: connection reset by peer; retrying
E0609 08:31:53.215384   24243 scheduler.go:501] Internal error binding pod: (Post http://127.0.0.1:58132/api/v1/namespaces/sample/pods/sched-perf-pod-nxk5h/binding: read tcp 127.0.0.1:62867->127.0.0.1:58132: read: connection reset by peer)
E0609 08:31:53.215396   24243 scheduler.go:501] Internal error binding pod: (Post http://127.0.0.1:58132/api/v1/namespaces/sample/pods/sched-perf-pod-gv6gs/binding: read tcp 127.0.0.1:62860->127.0.0.1:58132: read: connection reset by peer)

```
2. A panic occurs： integer divide by zero

```
I0609 08:32:05.261215   24243 util.go:53] destroyed API server
--- FAIL: TestSchedule100Node3KPods (28.83s)
panic: runtime error: integer divide by zero [recovered]
	panic: runtime error: integer divide by zero

goroutine 99 [running]:
testing.tRunner.func1(0xc420c760f0)
	/usr/local/Cellar/go/1.10.2/libexec/src/testing/testing.go:742 +0x29d
panic(0x4077020, 0x7aa0ba0)
	/usr/local/Cellar/go/1.10.2/libexec/src/runtime/panic.go:502 +0x229
k8s.io/kubernetes/test/integration/scheduler_perf.schedulePods(0xc424c3f980, 0x0)
	/Users/wangguoliang/Documents/mos/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/integration/scheduler_perf/scheduler_test.go:166 +0x7c6
k8s.io/kubernetes/test/integration/scheduler_perf.TestSchedule100Node3KPods(0xc420c760f0)
	/Users/wangguoliang/Documents/mos/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/integration/scheduler_perf/scheduler_test.go:78 +0x86
testing.tRunner(0xc420c760f0, 0x497a1f8)
	/usr/local/Cellar/go/1.10.2/libexec/src/testing/testing.go:777 +0xd0
created by testing.(*T).Run
	/usr/local/Cellar/go/1.10.2/libexec/src/testing/testing.go:824 +0x2e0
exit status 2
FAIL	k8s.io/kubernetes/test/integration/scheduler_perf	28.949s
!!! Error in ./test-performance.sh:51
  Error in ./test-performance.sh:51. 'go test -test.run=. -test.timeout=60m -test.short=false' exited with status 1
Call stack:
  1: ./test-performance.sh:51 main(...)
Exiting with status 1
~/Documents/mos/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/integration/scheduler_perf
+++ [0609 08:32:05] performance test cleanup complete
```

**What you expected to happen**:
Should be able to run successfully many times in succession./test-performance.sh

**How to reproduce it (as minimally and precisely as possible)**:
You can run ./test-performance.sh multiple times consecutively.

**Anything else we need to know?**:

**Environment**:
Darwin  16.7.0 Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64 x86_64

- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/test/integration/scheduler_perf/scheduler_test.go
+++ b/test/integration/scheduler_perf/scheduler_test.go
@@ -162,8 +162,12 @@ func schedulePods(config *testConfig) int32 {
 		// return the worst-case-scenario interval that was seen during this time.
 		// Note this should never be low due to cold-start, so allow bake in sched time if necessary.
 		if len(scheduled) >= config.numPods {
+			consumed := int(time.Since(start) / time.Second)
+			if consumed <= 0 {
+				consumed = 1
+			}
 			fmt.Printf("Scheduled %v Pods in %v seconds (%v per second on average). min QPS was %v\n",
-				config.numPods, int(time.Since(start)/time.Second), config.numPods/int(time.Since(start)/time.Second), minQps)
+				config.numPods, consumed, config.numPods/consumed, minQps)
 			return minQps
 		}
 

